examples/README.md

```diff
@@ -1,32 +1,29 @@
-# Remix Examples
+# Redis Session Storage Using Upstash Example
```

Comment:
These changes should be reverted and entered in `examples/redis-upstash-session/README.md`

Reply:
Sorry for that.